### DISCUSSION TITLE
Invalid user, while using mention in android client

### DISCUSSION THREAD

**Discussion by DeedsBaron** [bwmarrin/discordgo#1164] [Q&A] [answered]
Hello again, i’ve problem, trying to use in response message `i.Member.Mention()`, where `i *discordgo.InteractionCreate `, everything is ok on pc, and ios app, but on android i ve invalid user, am i doing smth wrong and is there any chance to fix this or is it android client bug, big thx) 
![FC34D769-90D6-4C9D-A400-FBE9B7E16DE4](https://user-images.githubusercontent.com/80648065/163730618-d4b0dc05-f4e5-4ebd-a29c-8ba70f0630ba.jpeg)

**Comment by 42atomys** [comment, marked as answer]
Hi @DeedsBaron, this is a mobile client bug due to a performance trick by Discord. Something this bug is also displayed on iOS with a number like `<@18431843184159641>` but mentions is correctly set and visible on desktop and web versions.

TL;DR: Not related to discordgo :)
